src/OrchardCore.Modules/OrchardCore.Settings/SiteSettings.cs

```diff
@@ -24,7 +24,7 @@ public class SiteSettings : Entity, ISite
         public string SiteSalt { get; set; }
         public string PageTitleFormat { get; set; }
         public string SuperUser { get; set; }
-        public bool UseCdn { get; set; } = true;
+        public bool UseCdn { get; set; } = false;
```

Comment:
```csharp
public bool UseCdn { get; set; }
```

No need to set `false`

Reply:
Ha, you're right, I was so focused on solving the problem that I forgot the default value